global_settings/wagtail_hooks.py

```diff
@@ -36,12 +36,20 @@ def register_strikethrough_feature(features):
 
 @hooks.register('after_edit_page')
 def purge_cloudfront_caches(page, request):
-    try:
-        distribution = CloudfrontDistribution.objects.all()[0]
-        cf = boto.connect_cloudfront()
-        cf.create_invalidation_request(distribution.distribution_id, [])
-    except: #TODO: This is a broad exception - but we are having issues with the cache. Will tighten down once we start using it.
-        pass
+    distribution = CloudfrontDistribution.objects.all()[0]
+    client = boto3.client('cloudfront')
+    response = client.create_invalidation(
+        DistributionId=distribution.distribution_id,
+        InvalidationBatch={
+            'Paths': {
+                'Quantity': 1,
+                'Items': [
+                    'apps/cms/api/*'
```

Comment:
i think you want the leading slash here

Reply:
nvm the docs says its optional https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Invalidation.html we use the leading slash in other places though